source/parser.hera

```diff
@@ -5049,11 +5049,19 @@ LexicalDeclaration
     }
   # NOTE: Added const shorthand
   # NOTE: Using ExtendedExpression to allow for If/SwitchExpressions
-  InsertConst ( BindingPattern / BindingIdentifier ) TypeSuffix? __ ConstAssignment ExtendedExpression ->
+  InsertConst ( BindingPattern / BindingIdentifier ) TypeSuffix? __ ConstAssignment ExtendedExpression:expression ( _? PostfixStatement )?:post ->
```

Comment:
Is there a reason you can't just replace `ExtendedExpression` with `PostfixedExpression`?

Reply:
Just didn't think of it, it's much better that way, thanks!